pkg/pipelines/bootstrap.go

```diff
@@ -45,9 +45,9 @@ const (
 	rolesPath             = "02-rolebindings/pipeline-service-role.yaml"
 	rolebindingsPath      = "02-rolebindings/pipeline-service-rolebinding.yaml"
 	serviceAccountPath    = "02-rolebindings/pipeline-service-account.yaml"
-	secretsPath           = "03-secrets/gitops-webhook-secret.yaml"
-	authTokenPath         = "03-secrets/git-host-access-token.yaml"
-	basicAuthTokenPath    = "03-secrets/git-host-basic-auth-token.yaml"
+	secretsPath           = "03-secrets/gitops-webhook-secret.yaml"     //nolint:gosec
+	authTokenPath         = "03-secrets/git-host-access-token.yaml"     // nolint:gosec
+	basicAuthTokenPath    = "03-secrets/git-host-basic-auth-token.yaml" // nolint:gosec
```

Comment:
If there are gosec warnings, can they be fixed?

Reply:
I think gosec finds things like `token` and `secret` in names.

Alternative name suggestions that don't include `secret` or `token` welcome...